README.md

```diff
@@ -1,2 +1,6 @@
 # [Sample Repo] PR Practice
 A sample repo for practicing how to create Pull Requests
+# Introduction
+Creating Pull Requsts (PRs) is needed when using RCS in a multi-person projects.
+This repo can be used to practice creating PRs.
+---> this line added after PR is submitted, commit and push the changes, see if changes reflected in PR
```

Comment:
commit AFTER PRs is submitted will be automatically reflected, as long as you push it here